docs/contribute/_snippets/changelog-fields.md

```diff
@@ -0,0 +1,103 @@
+
+```yaml
+##### Required fields #####
+title:
+
+# A required string that is a short, user-facing headline
+# Max 80 characters
+
+type: 
+
+# A required string that contains the type of change.
+# It can be one of:
```

Comment:
Only one? Why using a YAML list for this then? Or if the list is just for presentation purpose here, I would show an example of how to fill out the field.

Reply:
Yes, each changelog is a YAML file that can pertain to only one type of release item.
The goal of this snippet was to show the fields that can appear in the file... it doesn't get used anywhere else and IMO shouldn't be the source of truth for the acceptable values. I think from these comments it sounds like I should pare down the snippet.